include/quic/endpoint.hpp

```diff
@@ -53,7 +47,14 @@ namespace oxen::quic
         Endpoint(Endpoint&&) = delete;
         Endpoint& operator=(Endpoint&&) = delete;
 
-        explicit Endpoint(Network& n, const Address& listen_addr);
+        std::string local_addr() { return _local.to_string(); }
```

Comment:
I think this method was added for debug, but isn't really needed: if someone wants it as a string they have a few options: `ep.local().to_string()` or `"{}"_format(ep.local())`, or even just pass `ep.local()` if it's already going into a formattable string.